indexing-service/src/main/java/io/druid/indexing/common/task/IndexTask.java

```diff
@@ -1485,6 +1485,7 @@ public long getMaxBytesInMemory()
     }
 
     @JsonProperty
+    @Override
```

Comment:
Please add `@Nullable` here too.

Reply:
Done (but line changed is below this comment so github isn't squashing this one in the UI)